lib/modules/platform/gerrit/client.ts

```diff
@@ -103,15 +104,17 @@ class GerritClient {
     });
   }
 
-  async updateCommitMessage(
+  async updateChangeSubject(
     number: number,
-    gerritChangeID: string,
-    prTitle: string,
+    currentMessage: string,
+    newSubject: string,
   ): Promise<void> {
-    await this.setCommitMessage(
-      number,
-      `${prTitle}\n\nChange-Id: ${gerritChangeID}\n`,
+    // Replace first line of the commit message with the new subject
+    const newMessage = currentMessage.replace(
+      new RegExp(`^.*$`, 'm'),
```

Comment:
```suggestion
      regEx(`^.*$`, 'm'),
```

use our regex helper